README.md

```diff
@@ -63,32 +67,34 @@ The following steps are only relevant if you intend to contribute to the reposit
 
 - Install development dependencies (optionally skipping Lockfile creation):
 
-	  pipenv install --dev --skip-lock
+	    pipenv install --dev --skip-lock
 
 - Install package in development mode (this step replaces `pip install aea[all]` above):
 
-	  pip install -e .[all]
+	    pip install -e .[all]
```

Comment:
see above:

- Before

	  pip install -e .[all]

- After

	    pip install -e .[all]

Reply:
Yes that's on purpose :)

Reply:
Can you explain the rationale behind that?

In my opinion, it should be without space. Both for an aesthetic reason and a practical reason. For the latter, if I wanted to copy & paste the command on the terminal, I have to remove the spaces manually.